CHANGELOG.md

```diff
@@ -331,6 +331,9 @@ Read the [SwiftGen 6.0 Migration Guide](Documentation/MigrationGuide.md#swiftgen
   [SwiftGen/templates#84](https://github.com/SwiftGen/templates/pull/84)
   [Txai Wieser](https://github.com/txaiwieser)
   [SwiftGen/templates#81](https://github.com/SwiftGen/templates/pull/81)
+* Templates to generate ObjC source for .strings.  
```

Comment:
Add "For more information, check the template's documentation." with a link, see other entries.

Reply:
I think also I should be moving this out of 'Version 5.3.0' and into ... which version?

Reply:
Good catch, hadn't even noticed it was in the wrong section 👍